wagtail/test/dummy_external_storage.py

```diff
@@ -26,11 +27,16 @@ def _open(self, name, mode="rb"):
         # File object
         return DummyExternalStorageFile(open(self.wrapped.path(name), mode))
 
-    # Wrap all other functions
-
     def _save(self, name, content):
+        file_pos = content.tell()
+        if file_pos != 0:
+            raise ValueError(
+                "content file pointer should be at 0 - got %d instead" % file_pos
```

Comment:
Nit: to be consistent with other errors that we raise:

```suggestion
                "Content file pointer should be at 0 - got %d instead" % file_pos
```